spec/unit/puppet/provider/dsc_base_provider/dsc_base_provider_spec.rb

```diff
@@ -1541,6 +1585,12 @@
           expect(date_time).to receive(:format).and_return('2100-01-01')
           expect(result).to match(/datetime = \[DateTime\]'2100-01-01'/)
         end
+
+        context 'When a'
+        it 'casts the formatted timestamp string to DateTime in the property hash' do
+          expect(date_time).to receive(:format).and_return('2100-01-01')
+          expect(result).to match(/datetime = \[DateTime\]'2100-01-01'/)
+        end
```

Comment:
This looks like it ended up in the wrong place?

Reply:
oops